### DISCUSSION TITLE
Setting column order on a board

### DISCUSSION THREAD

**Discussion by malevans** [marcusolsson/obsidian-projects#748] [Help] [answered]
Apologies up front if I am missing something obvious but I don't seem to be able to control the order of the columns on a board. In my test environment I have a basic Kanban style configured using the following statuses for columns:
To-do
Doing
Done

But when I create a board view the columns are presented in alphabetical order:
Doing
Done
To-do

which breaks the traditional left-to-right flow.

Appreciate any comments to help point me in the right direction.
Cheers

**Comment by Acylation** [comment, marked as answer]
Hi! The drag and drop feature is available for columns so that you can easily reorder them. Feel free to follow up if this doesn't meet your need.
